frontend/src/custom.d.ts

```diff
@@ -3,3 +3,8 @@ declare module '*.svg' {
     const content: any
     export default content
 }
+// This fixes TS errors when importing a .png file
+declare module '*.png' {
```

Comment:
NIT: There any way you can use all svgs for the icons? I assume you've tried looking but it'd be nicer for consistency

Reply:
Yea unfortunately your assumption is correct – I think that we'll do a design pass over this if we decide to keep in app and can redo icons at that point